src/app/pages/flex-page/blocks/RichTextBlock.scss

```diff
@@ -17,6 +17,7 @@
 
   p {
     @include body-font(1.6rem);
+    line-height: 2.5rem;
```

Comment:
This is the same as `@include set-font(body-regular)`, which is the usual text block stuff on osweb.
There's also `@include set-font(body-large)` which has 1.8rem font and line-height of 3rem.